ports/stm32/Makefile

```diff
@@ -251,6 +251,7 @@ SRC_C += \
 	pyb_can.c \
 	usb.c \
 	eth.c \
+	phy.c \
```

Comment:
I suggest calling this new file `eth_phy.c`.

Reply:
Done, in contrast to usb_phy.*